repository-data/webfiles/src/main/resources/site/src/scss/rebrand/components/_api-catalogue.scss

```diff
@@ -50,11 +50,13 @@
     }
   }
 
-  input.section-folder:not(:checked) {
-
-    ~ label .icon svg {
-      rotate: -90deg;
+  input.section-folder:checked {
+    ~ .nhsd-m-filter-menu-section__menu-button label .icon svg {
```

Comment:
i would just write `~.nhsd-m-filter-menu-section__menu-button svg` here, it isn't necessary to target each subsequent child when we only have one svg in the div anyway

Reply:
Indeed. Fixing.